elements/bulbs-video/components/revealed.js

```diff
@@ -107,7 +105,8 @@ export default class Revealed extends React.Component {
   }
 
   makeVideoPlayer (element, playerOptions) {
-    new VideoPlayer(element, playerOptions); // eslint-disable-line no-new
+    let player = new VideoPlayer(element, playerOptions);
+    player.player.play();
```

Comment:
Does this imply every single video carousel (and any bulbs-video implementation would be auto-play?  We don't want to do that for all players if so -- just on a one by one basis, preferably passed in with the autoplay HTML attribute